src/bun.js/node/zlib/NativeBrotli.zig

```diff
@@ -179,6 +180,15 @@ const Context = struct {
     }
 
     pub fn reset(this: *Context) Error {
+        if (this.state != null) {
```

Comment:
Add a deinitState() method that doesn't set mode and then reuse that code in both call-sites